recipes/toil/meta.yaml

```diff
@@ -0,0 +1,124 @@
+# copied from https://github.com/bioconda/bioconda-recipes/blob/9c08a16bac9b7d0a076578823d5008b8bbfc466e/recipes/toil/meta.yaml
+{% set name = "toil" %}
+{% set version = "7.0.0" %}
+{% set sha256 = "c3f4e9567754f5ad6fe56b90913b7c1c9aaade32b545502a483cd57f70f24777" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --no-cache-dir -vvv
+  run_exports:
+    - {{ pin_subpackage(name, max_pin="x") }}
+  entry_points:
+    - toil = toil.utils.toilMain:main
+    - _toil_worker = toil.worker:main
+    - toil-cwl-runner = toil.cwl.cwltoil:main [cwl]
+    - toil-wdl-runner = toil.wdl.wdltoil:main [wdl]
```

Comment:
I'm not familiar with the `[]` syntax for entrypoints. What does that do?

Reply:
I believe it is coming from `setuptools` syntax for defining `extras` dependency: https://github.com/DataBiosphere/toil/blob/f71dc86c57680e9dfa78e7d283747dfbc04eb362/setup.py#L119C1-L128C92

Since, `conda install` does not work that way, this can probably be dropped. Your thoughts?

Reply:
Yep. That won't work. You will need to include those extras or create a multiple outputs recipe for them.